tensorflow/core/common_runtime/mkl_layout_pass.cc

```diff
@@ -394,7 +394,7 @@ class MklLayoutRewritePass : public GraphOptimizationPass {
          CopyAttrsAll, RewriteIfAtleastOneMklInput, GetRewriteCause()});
     rinfo_.push_back({csinfo_.avg_pool,
                       mkl_op_registry::GetMklOpName(csinfo_.avg_pool),
-                      CopyAttrsAll, AlwaysRewrite, GetRewriteCause()});
+                      CopyAttrsAll, CheckRewrite, GetRewriteCause()});
```

Comment:
I think `RewriteIfX86` is a more meaningful name.
```suggestion
                      CopyAttrsAll, RewriteIfX86, GetRewriteCause()});
```

Reply:
Thank you for reviewing this PR. I have made the necessary changes and uploaded it.